src/build.md

```diff
@@ -1,3 +1,69 @@
+
+## The Two Modalities of the SORA Builders Programme
+
+The SORA Builders Programme is an extension of the [SORA RFP program](rfp.md). The SORA Builders Programme branches out into two modalities, **New Project (Incubation)** and **Existing Project (Port).** As the names imply, this programme will support the implementation and development of new projects, as well as porting existing apps onto the SORA network. 
+
+Both new projects and ports can receive compensation in the form of XOR, as well as development support from the SORA community. Because effectively time and development effort are not free, and in order to uphold the community standards, not everyone who applies will be accepted by the programme.
+
+It is also important to mention that depending on the value added by the project, it may also qualify for [Social Insurance](https://medium.com/sora-xor/social-insurance-for-systematically-important-infrastructure-18a63ef711ca), however, there will be required due diligence and the project will have to prove systematic importance to the community.
+
+It is easy to ask *wot* and *wen* when it comes to development, but *putting your RFP where your complaints are* is not that simple to do. The community has no limitation to what is proposed, however, [*spam](https://youtu.be/duFierM1yDg), CEX requests,* and other off-topic requests will not be considered.
+
+## Participating in the SORA Builders Programme
+
+- To join the SORA Builders Programme, please [fill this
+form](https://forms.gle/1cdvKTHLXnBXbMTr9), then go to the [SORA
+Builders Telegram channel](https://t.me/sorabuilders) in telegram.
+
+- After you have submitted your proposal, it will be reviewed
+  internally and you will be invited to talk to the developers, if it
+  is a match, the process will continue.
+  
+  - **Milestones** Funding support provided by the SORA Builders
+  Programme is subject to the completion of milestones presented
+  within the proposal. 
+  - **Developer Support** Depending on the milestones, your team can
+    request support from the developers to help porting or building
+    their project within the SORA network.
+	- **Co-Marketing** Part of the support provided from the SORA
+  Builder's Programme includes amplification of the project in the
+  SORA Ecosystem Updates and on SORA social media channels.
+  
+- **Graduation** Once your project is live on SORA, you will be part
+  of the elite SORA Builders Graduates. This doesn't mean that the
+  support provided will be over. You can still reach out to the tech
+  team for any support required. Additionally, if your proejct
+  qualifies and you chose to sign up
+  for Social Insurance for [Systemically Important Infrastructure
+  ](social-insurance.md), the governance of your app will be delegated
+  to the SORA Parliament. 
```

Comment:
is there a reason this info is in an unordered list? I also can't tell is the indentation is intentional or not